deployment/Dockerfile-node

```diff
@@ -1,19 +1,18 @@
-FROM debian:bookworm-slim@sha256:acd98e6cfc42813a4db9ca54ed79b6f702830bfc2fa43a2c2e87517371d82edb
+# FROM debian:bookworm-slim@sha256:acd98e6cfc42813a4db9ca54ed79b6f702830bfc2fa43a2c2e87517371d82edb
+FROM google/cloud-sdk:debian_component_based
```

Comment:
Note for posterity: We need this image to support the `gcloud` command in `start.sh`